src/D2L.Bmx/Okta/OktaSessionStorage.cs

```diff
@@ -0,0 +1,54 @@
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using D2L.Bmx.Okta.Models;
+
+namespace D2L.Bmx.Okta;
+
+internal class OktaSessionStorage {
+
+	internal static void SaveSessions( List<OktaSessionCache> sessions ) {
+
+		string jsonString = JsonSerializer.Serialize(
+			sessions.ToArray(),
```

Comment:
do we have to convert to array? list doesn't work with source generator?

Reply:
Looks like it does work 😄 . I originally was just passing around array but realized list would be easier and then forgot to try changing the serializable type